test/reduce.test.js

```diff
@@ -177,6 +177,304 @@ const assertComparisonNotReduced = (t, comparison, value = 'test') => {
   t.deepEqual(reduce(policy, { user }), expected);
 };
 
+const getUserCustomConditions = (resourceName) => {
+  return {
+    'user.customAttributes.secret': {
+      comparison: 'suffixOf',
+      target: `${resourceName}.secret`,
+    },
+    'user.customAttributes.id': {
+      comparison: 'equals',
+      target: `${resourceName}.id`,
+    },
+    'user.customAttributes.isActive': {
+      comparison: 'exists',
+      target: `${resourceName}.isActive`,
+    },
```

Comment:
I don't think this is a valid rule. `exists` should not be used with a `target` or `value`, I think.

Maybe we need a special case?

https://phc.docs.lifeomic.com/development/abac-syntax#exists

Reply:
Good catch.